src/Form/MiradorConfigForm.php

```diff
@@ -39,13 +39,26 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['mirador_library_fieldset']['mirador_library_installation_type'] = [
       '#type' => 'radios',
       '#options' => [
-        'local'=> $this->t('Local library placed in /libraries inside your webroot.'),
         'remote' => $this->t('Default remote location'),
+        'local'=> $this->t('Local library placed in /libraries inside your webroot.'),
+
       ],
       '#description' => $this->t("For local, put the output of 'npm run webpack' of <a href=\"https://github.com/roblib/mirador-integration-islandora\">Mirador Integration Islandora</a> into web/library/mirador/dist/ and ensure it's named main.js."),
       '#default_value' => $config->get('mirador_library_installation_type'),
     ];
 
+    $form['mirador_library_fieldset']['mirador_library_minified'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Local library is minified'),
+      '#description' => $this->t("Check this if the local library has been minified."),
+      '#default_value' => $config->get('mirador_library_minified'),
```

Comment:
If nullable, this should deal with a default consistently.

Reply:
Okay? But say one is submitting this form for any other arbitrary reason? Now, all of a sudden, they're explicitly specifying a value in config that they might now not want, if they wanted to keep things with the null/default behavior?

Reply:
I threw the hook in.  It won't hurt, but I don't think it will actually change behavior anywhere.  minified NULL and minified false are treated the same,